### DISCUSSION TITLE
Can't resize preview pane by more than 50%

### DISCUSSION THREAD

**Discussion by GuildOfCalamity** [KDE/ghostwriter#882] [General]
Could you bring this feature back? I am unable to resize the preview pane over 50% from the raw markup pane.

**Comment by olberger** [comment]
Is this a bug or feature ? I noticed a colleague running Ubuntu could resize at will, while me on Debian can't... but it isn't clear what the difference is between our tools :-/

**Comment by GuildOfCalamity** [comment]
I'm not sure what the intent was but, the previous version would allow you to go past 50%. Maybe he could add an option in the settings?
